lib/json_schemer.rb

```diff
@@ -59,22 +59,25 @@
 require 'json_schemer/openapi30/vocab/base'
 require 'json_schemer/openapi30/vocab'
 require 'json_schemer/openapi'
+require 'json_schemer/configuration'
 require 'json_schemer/schema'
 
 module JSONSchemer
-  class UnsupportedMetaSchema < StandardError; end
-  class UnsupportedOpenAPIVersion < StandardError; end
-  class UnknownRef < StandardError; end
-  class UnknownFormat < StandardError; end
-  class UnknownVocabulary < StandardError; end
-  class UnknownContentEncoding < StandardError; end
-  class UnknownContentMediaType < StandardError; end
-  class UnknownOutputFormat < StandardError; end
-  class InvalidRefResolution < StandardError; end
-  class InvalidRefPointer < StandardError; end
-  class InvalidRegexpResolution < StandardError; end
-  class InvalidFileURI < StandardError; end
-  class InvalidEcmaRegexp < StandardError; end
+  UnsupportedMetaSchema = Class.new(StandardError)
+  UnsupportedOpenAPIVersion = Class.new(StandardError)
+  UnknownRef = Class.new(StandardError)
+  UnknownFormat = Class.new(StandardError)
+  UnknownVocabulary = Class.new(StandardError)
+  UnknownContentEncoding = Class.new(StandardError)
+  UnknownContentMediaType = Class.new(StandardError)
+  UnknownOutputFormat = Class.new(StandardError)
+  UnknownRegexpResolver = Class.new(StandardError)
+  UnknownAccessMode = Class.new(StandardError)
```

Comment:
@davishmcclurg Added these 2 new errors for some validations in the `Configuration` class